packages/react-server-chrome-extension-timeline/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "react-server-chrome-extension-timeline",
+  "version": "1.0.0",
+  "description": "This extension will display the timeline for react-server applications",
+  "main": "popup.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
This is causing the Travis CI tests to fail.  Please add automated tests to this package so that we can keep it up to date!